sales_page/index.html

```diff
@@ -6,20 +6,20 @@
 </head>
 <body>
   <div id="sales">
-    <article
+    <article>
       <form id="addSalesForm">
         <fieldset>
-          <legend>Sales information</legend>
-          <lable>Location</label>
-            <input location = "location" type = "text"/>
-          <label>Avg cookies</label>
-            <input avgCookies = "avgCookies" type = "text"/>
-          <label>min Customer</label>
-            <input minCustomer = "minCustomer" type = "text"/>
-          <label>max Customer</label>
-            <input maxCustomer = "maxCustomer" type = "text"/>
+            <legend>Sales information</legend>
+            <lable>Location</label>
+            <input name = "location" type = "text"/>
```

Comment:
Do you mean to validate the input for the numbers are indeed numbers? I validate the location using javascript. Would you recommend all of them in the same html file? Please let me know. Thanks!  Also thanks for the link.